_monitoring-your-cluster/pa/rca/shard-hotspot.md

```diff
@@ -0,0 +1,102 @@
+"HotShardClusterRca": [{
+  "rca_name": "HotShardClusterRca",
+  "timestamp": 1680721367563,
+  "state": "unhealthy",
+  "HotClusterSummary": [
+    {
+      "number_of_nodes": 3,
+      "number_of_unhealthy_nodes": 1,
+      "HotNodeSummary": [
+        {
+          "node_id": "7kosAbpASsqBoHmHkVXxmw",
+          "host_address": "192.168.80.4",
+          "HotResourceSummary": [
+            {
+              "resource_type": "cpu usage",
+              "resource_metric": "cpu usage(num of cores)",
+              "threshold": 0.027397981341796683,
+              "value": 0.034449630200405396,
+              "time_period_seconds": 60,
+              "meta_data": "ssZw1WRUSHS5DZCW73BOJQ index9 4"
+            },
+            {
+              "resource_type": "heap",
+              "resource_metric": "heap alloc rate(heap alloc rate in bytes per second)",
+              "threshold": 7605441.367010161,
+              "value": 10872119.748328414,
+              "time_period_seconds": 60,
+              "meta_data": "ssZw1WRUSHS5DZCW73BOJQ index9 4"
+            },
+            {
+              "resource_type": "heap",
+              "resource_metric": "heap alloc rate(heap alloc rate in bytes per second)",
+              "threshold": 7605441.367010161,
+              "value": 8019622.354388569,
+              "time_period_seconds": 60,
+              "meta_data": "QRF4rBM7SNCDr1g3KU6HyA index9 0"
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}]
+```
+
+## Response fields
+
+The following table lists the response fields.
+
+Field | Type | Description
+:--- | :--- | :---
+rca_name | String | The name of the RCA. In this case, "HotShardClusterRca".
+timestamp | Integer | The timestamp of the RCA.
+state | Object | The state of the cluster determined by the RCA. The `state` can be `healthy`, `unhealthy`, or `unknown`.
+HotClusterSummary.HotNodeSummary.number_of_nodes | Integer | The number of nodes in the cluster.
+HotClusterSummary.HotNodeSummary.number_of_unhealthy_nodes | Integer | The number of nodes found to be in an `unhealthy` state.
+HotClusterSummary.HotNodeSummary.HotResourceSummary.resource_type | Object | The type of resource checked, either "cpu usage" or "heap".
+HotClusterSummary.HotNodeSummary.HotResourceSummary.resource_metric | String | The definition of the resource_type. Either "cpu usage(num of cores)" or "heap alloc rate(heap alloc rate in bytes per second)".
+HotClusterSummary.HotNodeSummary.HotResourceSummary.threshold | Float | The value that determines if a resource is highly utilized.
+HotClusterSummary.HotNodeSummary.HotResourceSummary.value | Float | The current value of the resource.
+HotClusterSummary.HotNodeSummary.HotResourceSummary.time_period_seconds | Time | The amount of time that the resource_type has to be above the threshold value in order to mark the shard state as `unhealthy`.
+HotClusterSummary.HotNodeSummary.HotResourceSummary.meta_data | String | The metadata associated with the resource_type.
```

Comment:
This is the most important field. We can break it down. 
`"meta_data": "QRF4rBM7SNCDr1g3KU6HyA index9 0"`, the value here is made of 3 fields representing `Node_Name Index_Name Shard_Id` respectively.  The Shard `0` of Index `index9` on Node `QRF4rBM7SNCDr1g3KU6HyA` is hot.